src/redux/actions/crudActions/getUsersActions.js

```diff
@@ -0,0 +1,24 @@
+import { crudConstants } from "../constants";
+import { userServiceHandler } from "../../../utils/userHandler";
+import { alertActions } from "../alertActions";
+
+export const getAll = () => {
+  const success = (users) => {
```

Comment:
it might be a good idea to include a case for when we are requesting the list of users from the BE